File: src/main/java/br/com/jiratorio/domain/response/BoardDetailsResponse.java
package br.com.jiratorio.domain.response;

import br.com.jiratorio.domain.vo.DynamicFieldConfig;
import br.com.jiratorio.domain.ImpedimentType;
import java.util.List;
import lombok.Data;
import lombok.ToString;

@Data
@ToString(of = "name")
public class BoardDetailsResponse {

    private Long externalId;

    private String name;

    private String startColumn;

    private String endColumn;

    private List<String> fluxColumn;

    private List<String> ignoreIssueType;

    private String epicCF;

    private String estimateCF;

    private String systemCF;

    private String projectCF;

    private Boolean calcDueDate;

    private Boolean ignoreWeekend;

    private ImpedimentType impedimentType;

    private List<String> impedimentColumns;

    private List<DynamicFieldConfig> dynamicFields;

}

File: src/main/java/br/com/jiratorio/domain/response/BoardDetailsResponse.kt
package br.com.jiratorio.domain.response

import br.com.jiratorio.domain.dynamicfield.DynamicFieldConfig
import br.com.jiratorio.domain.impediment.ImpedimentType

data class BoardDetailsResponse(
    var externalId: Long? = null,
    var name: String? = null,
    var startColumn: String? = null,
    var endColumn: String? = null,
    var fluxColumn: List<String>? = null,
    var ignoreIssueType: List<String>? = null,
    var epicCF: String? = null,
    var estimateCF: String? = null,
    var systemCF: String? = null,
    var projectCF: String? = null,
    var ignoreWeekend: Boolean? = null,
    var impedimentType: ImpedimentType? = null,
    var impedimentColumns: List<String>? = null,
    var dynamicFields: List<DynamicFieldConfig>? = null,
    var dueDateCF: String? = null,
    var dueDateType: String? = null
)
